cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/IamCookieInterceptor.java

```diff
@@ -32,9 +32,10 @@
     public static final String IAM_TOKEN_SERVER_URL_PROPERTY_KEY = "com.cloudant.client.iamserver";
     public final URL iamServerUrl;
     private final byte[] tokenRequestPayload;
+    private final URL proxyURL;
 
-    public IamCookieInterceptor(String apiKey, String baseUrl) {
-        super(baseUrl, "/_iam_session", null);
+    public IamCookieInterceptor(String apiKey, String baseUrl, URL proxyURL) {
+        super(baseUrl, "/_iam_session", null, proxyURL);
```

Comment:
To avoid breaking compatibility I'd prefer to see this retain the original ctor and add a second ctor for the `proxyURL` case in the same way as the `CookieInterceptor`.

Reply:
Fixed in a1c95fad4ccf0e845fe4649613fc184c33168220